templates/step/edit.html.ep

```diff
@@ -15,6 +15,14 @@
     % }
     setup_needle_editor( <%= b(encode_json($default_needle)) %> );
     setupResultButtons();
+	% for my $pro (@$properties) {
```

Comment:
In Perl we always use 4 spaces. Since the templates are not tidied automatically you need to take care manually line with the JavaScript files.